stack_definition.json

```diff
@@ -67,6 +67,20 @@
       "type": "boolean",
       "hidden": false,
       "default": false
+    },
+    {
+      "name": "resource_groups_scope",
+      "required": false,
+      "type": "list",
+      "hidden": false,
+      "default": []
+    },
+    {
+      "name": "attachment_schedule",
+      "required": false,
+      "type": "string",
+      "hidden": false,
+      "default": "daily"
```

Comment:
We'd better set default to "monthly". This will trigger a first scan right away, but will not rake up daily cost.